src/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkImageBrowser.tsx

```diff
@@ -28,23 +29,51 @@ export const ArtworkImageBrowser: React.FC<ArtworkImageBrowserProps> = ({
     max: figures.length,
   })
 
-  const images = compact(artwork.images)
-  const hasGeometry = !!images[0]?.width
-  const maxHeight = Math.max(
-    ...images.map(image => {
-      const scaled = scale({
-        width: image.width!,
-        height: image.height!,
-        maxWidth: MAX_DIMENSION,
-        maxHeight: MAX_DIMENSION,
+  // Here we pre-emptively scale the figures to figure out which is going
+  // to have the largest height. We use this to set a fixed height on the container
+  // so that the UI doesn't shift around.
+  const maxHeight = useMemo(() => {
+    return Math.max(
+      ...figures.map(figure => {
+        switch (figure.__typename) {
+          case "Image": {
+            if (!figure.width || !figure.height) return MAX_DIMENSION
+
+            return scale({
+              width: figure.width,
+              height: figure.height,
+              maxWidth: MAX_DIMENSION,
+              maxHeight: MAX_DIMENSION,
+            }).height
+          }
+
+          case "Video": {
+            return scale({
+              width: figure.videoWidth,
+              height: figure.videoHeight,
+              maxWidth: MAX_DIMENSION,
+              maxHeight: MAX_DIMENSION,
+            }).height
+          }
+
+          default: {
+            return MAX_DIMENSION
+          }
+        }
       })
+    )
+  }, [figures])
 
-      return hasGeometry ? scaled.height : MAX_DIMENSION
-    })
-  )
+  const defaultIndex = useMemo(() => {
+    return (
+      figures.findIndex(figure => {
+        if (!("isDefault" in figure)) return false
+        return !!figure.isDefault
+      }) ?? 0
+    )
+  }, [figures])
```

Comment:
And actually; we should rename this since `isDefault` doesn't even make sense here. It should be like `viewInRoomIndex` or something.